underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -346,8 +355,14 @@ public boolean rename(String src, String dst) throws IOException {
       }
       // Rename each child in the src folder to destination/child
       String [] children = list(src);
-      for (String child: children) {
+      if (children == null) {
+        LOG.error("list({}) is returning null. "
+                + "Rename operation partially completed. Exiting...", src);
```

Comment:
Could you update this error message to be similar to the following one? ie. `Failed to list path {}, aborting rename.`